snakeviz/static/snakeviz.js

```diff
@@ -11,15 +11,16 @@ var sv_find_root = function sv_find_root (stats) {
     var diff = callers.subtract(callees);
     if (diff.size !== 0) {
```

Comment:
That was my original thought too (about diff.size == 1).  Nevertheless the set difference heuristic between callers and callees is still a useful heuristic for narrowing down the field.  So once you've already done it it's useful to go ahead with the difference set, rather than throwing it out and sorting *all* stats.